src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterApplicationServiceImplTest.java

```diff
@@ -256,4 +259,83 @@ void givenInvalidInstitutionId_whenCreatingAdminUserRegisterApplication_thenThro
         );
     }
 
+    @Test
+    void givenValidAdminUserRegisterApplicationId_whenAdminUserRegisterApplicationApproved_thenReturnNothing() {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+        AdminUserEntity mockAdminUserEntity = new AdminUserEntityBuilder()
+                .withValidFields()
+                .withStatus(AdminUserStatus.NOT_VERIFIED)
+                .build();
+        AdminUserRegisterApplicationEntity mockEntity = new AdminUserRegisterApplicationEntityBuilder()
+                .withValidFields()
+                .withStatus(AdminUserRegisterApplicationStatus.COMPLETED)
+                .withAdminUserId(mockAdminUserEntity.getId())
+                .withAdminUser(mockAdminUserEntity)
+                .build();
+
+        // When
+        Mockito.when(adminUserRegisterApplicationRepository.findById(mockId))
+                .thenReturn(Optional.of(mockEntity));
+        Mockito.when(adminUserRegisterApplicationRepository.save(Mockito.any(AdminUserRegisterApplicationEntity.class)))
+                .thenReturn(mockEntity);
+
+        // Then
+        adminUserRegisterApplicationService.approveRegistrationApplication(mockId);
+
+        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```